requirements.txt

```diff
@@ -27,3 +27,4 @@ boto
 django-storages
 django-htmlmin
 pyparsing
+bokeh==0.12.4
```

Comment:
Need to remove this line and add specify the version in `dropq_environment.yml`.

Reply:
OK.  Thanks for the helpful info.  But then what role does `requirements.txt` play in TaxBrain?

Reply:
This file is for installing packages from pip. It's mostly Django specific packages. Ideally we could be using the one environment file, and get rid of all these requirements files. `environment.yml` files can also install pip packages. This would be a good bit of code cleanup, but it would require some refactoring of the deployment scripts.

In the case of Bokeh, since its Continuum run, its best to install it from conda.

Reply:
@brittainhard,  Thanks for the explanation.